src/CP/core/search/strategies.jl

```diff
@@ -1,4 +1,77 @@
+"""SeaPearl Provides several pre-implemented search strategy to explore the search tree in a certain way. The specific strategy has to 
+be specified in the search! function. The search strategy choose how the _toCall_ Stack will be filled and empty ( ie. which branch will 
+be explored at a certain node ).
+"""    
 abstract type SearchStrategy end
 
-abstract type DFSearch <: SearchStrategy end
-abstract type ILDSearch <: SearchStrategy end
+struct DFSearch <: SearchStrategy end
+
+struct ILDSearch <: SearchStrategy end
```

Comment:
I switched back from type to struct as we needed to declare arguments for the different  ___RBSearch such as `L`.